apps/meteor/ee/app/livechat-enterprise/server/hooks/checkAgentBeforeTakeInquiry.ts

```diff
@@ -27,23 +27,20 @@ const validateMaxChats = async ({
 	};
 }) => {
 	if (!inquiry?._id || !agent?.agentId) {
-		cbLogger.debug('Callback with error. No inquiry or agent provided');
 		throw new Error('No inquiry or agent provided');
 	}
 	const { agentId } = agent;
 
 	if (!(await Livechat.checkOnlineAgents(undefined, agent))) {
-		cbLogger.debug('Callback with error. provided agent is not online');
 		throw new Error('Provided agent is not online');
 	}
 
 	if (!settings.get('Livechat_waiting_queue')) {
-		cbLogger.debug('Skipping callback. Disabled by setting');
 		return agent;
 	}
 
 	if (await allowAgentSkipQueue(agent)) {
-		cbLogger.debug(`Callback success. Agent ${agent.agentId} can skip queue`);
+		cbLogger.info(`Chat can be taken by Agent ${agentId}: agent can skip queue`);
```

Comment:
```suggestion
		cbLogger.debug(`Chat can be taken by Agent ${agentId}: agent can skip queue`);
```

Reply:
This is useful to know why a chat is still on queue and how often users hit limits.